packages/user-management-widget/src/lib/widget/state/selectors.ts

```diff
@@ -12,9 +13,18 @@ export const getUsersList = createSelector(getRawUsersList, (users) =>
   users.map((user) => ({
     ...user,
     status: userStatusMappings[user.status] || user.status,
+    roles: user.roleNames
+      .concat(user.userTenants?.[0]?.roleNames)
```

Comment:
We need the tenant ID, using 0 index is problematic

Reply:
Agreed. Related to pending issue that we need to resolve: https://github.com/descope/etc/issues/5601